.ci_support/migrations/cuda120.yaml

```diff
@@ -69,7 +68,7 @@ cuda_compiler:                 # [(linux or win64) and os.environ.get("CF_CUDA_E
   - cuda-nvcc                  # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
 
 cuda_compiler_version:         # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
-  - 12.1                       # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+  - 12                         # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
```

Comment:
As the migrator here is already vendored, have relaxed the CUDA compiler version to just pin the major (and not the minor version. This makes easier to customize in the recipe (as seen with `cuda-version`'s pinning in the `recipe/meta.yaml`'s `requirements` section)

Reply:
For some reason I totally missed this. This is neat and certainly better than having to touching this file every time we need to update build dependencies. This is basically fire-and-forget.

Reply:
And it works because we align `cuda-version` to match `cuda-python`'s `version` 

https://github.com/conda-forge/cuda-python-feedstock/blob/54a6579b17930f8184414e0c9212d29e561dd591/recipe/meta.yaml#L1-L4

https://github.com/conda-forge/cuda-python-feedstock/blob/54a6579b17930f8184414e0c9212d29e561dd591/recipe/meta.yaml#L39-L44

https://github.com/conda-forge/cuda-python-feedstock/blob/54a6579b17930f8184414e0c9212d29e561dd591/recipe/meta.yaml#L52-L61